### DISCUSSION TITLE
EventSource (SSE)

### DISCUSSION THREAD

**Discussion by szprytny** [pengzhanbo/vite-plugin-mock-dev-server#100] [Q&A] [answered]
Is there a way to mock server side events?

**Comment by pengzhanbo** [comment, marked as answer]
Mock `Server-Sent-Events` is supported starting from `v1.8.0`. See the [documentation](https://vite-plugin-mock-dev-server.netlify.app/en/guide/create-sse-stream) and [examples](https://vite-plugin-mock-dev-server.netlify.app/en/examples/event-source).
